reputation/reputation.go

```diff
@@ -252,7 +252,7 @@ func calculateScore(addr string, mi miner.IndexSnapshot, si faults.IndexSnapshot
 		askScore = 1
 	}
 
-	score := 50*faultsScore + 20*powerScore + 20*externalScore + 10*askScore
+	score := 50*faultsScore + 20*powerScore + 20*externalScore + 10*askScore + 1000*miner.ActiveDeals
```

Comment:
Bias the Reputation Module to also use new information in indexes.
The number of Active Deals for a miner is now a big weighting factor in testnet.